### ISSUE TITLE
[docs]: several images are missing alt text.

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#655]

**Comment by JesterOrNot** [comment]
I would also like to add that the images with ALT text lack [descriptive alt text](https://accessibility.psu.edu/images/alttext)

**Comment by nisarhassan12** [comment]
Thanks! I will work on this probably tomorrow and will try to make the ALT text descriptive for the images will go through all of the docs pages.
